.docker/ember-dev/Dockerfile

```diff
@@ -4,6 +4,6 @@ WORKDIR /app
 
 RUN apk add --no-cache nodejs-npm && \
     apk add --no-cache --virtual .build-deps git && \
-    npm install -g ember-cli
+    npm install -g ember-cli \
```

Comment:
Should make sure there is no trailing backslash, as it kills the Docker container build.
Of course, this image doesn't execute correctly anyway, so we don't really need to worry about this yet